.github/workflows/ci.yml

```diff
@@ -173,13 +173,3 @@ jobs:
         run: |
           ./dev-tools/build.sh
           PHP_CS_FIXER_TEST_ALLOW_SKIPPING_SMOKE_TESTS=0 vendor/bin/phpunit tests/Smoke/
-
-      - name: Execute deployment post-hook
```

Comment:
I wonder if we could still benefit from this logic if we would only update the secret var

Reply:
We can always bring it back, modify to our needs etc. It wasn't working, so wasn't needed.